mycity/mycity/intents/get_alerts_intent.py

```diff
@@ -82,7 +82,7 @@ def alerts_to_speech_output(alerts):
     for alert in alerts.values():
         all_alerts += alert + ' '
     if all_alerts.strip() == "":        # this is a kludgy fix for the {'alert header': ''} bug 
-        return "There are no alerts. City services are operating on their normal schedule."       
+        return "I see no alerts. City services are running on their normal schedules."       
```

Comment:
"I see no alerts." sounds like Alexa is not very confident versus saying "There are no alerts" or similar.

Reply:
Your point makes sense  - I was trying to make Alexa sound more friendly and human.